definitions/reports/compliance.rb

```diff
@@ -0,0 +1,30 @@
+module Checks
```

Comment:
These are all in the module Checks, but they live in `definitions/reports`. I think you should drop this line. If you want to differentiate checks from some other type for the report feature, then invert these, e.g.

```
module Report
  module Checks
```